### ISSUE TITLE
Circle issues on NewMap

### ISSUE THREAD

**Issue by moving-bits** [cgeo/cgeo#7842]
Just tried "circles" on OSM on an emulated device with Android 10 and today's master - great to have them back! Some regressions occurred, though:

- it looks like only traditional caches and caches with a manually set final get a circle drawn,
- and circles are slightly off-center

![image](https://user-images.githubusercontent.com/3754370/64883881-a5e58e00-d660-11e9-90a4-421431a459ab.png)

Google Maps v2 does show circles, but only after disabling and enabling - will create a separate issue for that one.

**Comment by Lineflyer** [comment]
Only waypoints assumes physical have circles.
So no circles for
- listing coords of mysteries
- virtual stages
- earth caches

**Comment by Lomanic** [comment]
From my testing with the current beta (2019.09.10-RC, so not using the NewMap I think), circles are centered exactly the same as your screenshot.

**Comment by Lineflyer** [comment]
I think its an optical illusion that they are not centered, due to the cache icon. The pinpoint of the icon however is the center of the circle. So everything fine?!

**Comment by moving-bits** [comment]
> Only waypoints assumes physical have circles.
> So no circles for
>     * listing coords of mysteries
>     * virtual stages
>     * earth caches

Ah, didn't know that - thanks for explaining. I'm mainly using NewMap, therefore haven't seen circles for a long time... ;-)

> I think its an optical illusion that they are not centered, due to the cache icon. The pinpoint of the icon however is the center of the circle. So everything fine?!

Yes, probably - will close this issue.
